recipes/kuliya/all/test_package/test_package.c

```diff
@@ -0,0 +1,8 @@
+#include <kuliya.h>
+
+int main()
+{
+    kuliya_init();
+    kuliya_schema *res = get_node_by_path("umkb/fst/dee/sec");
```

Comment:
Are these files from the `_data` folder meant to be packaged with the library for it to work? Maybe we should be checking the output of the function somehow?

Reply:
there is a `build.c` script to generate `data.h` file from that `_data` folder, so these are not read from that folder but instead they are static variables identified by these paths

Reply:
I will add assertion for the output